src/server/game/Battlefield/Zones/BattlefieldWG.h

```diff
@@ -1041,7 +1041,7 @@ const WintergraspTowerCannonData TowerCannon[WG_MAX_TOWER_CANNON] =
         2,
         {
             { 4448.138184f, 1974.998779f, 441.995911f, 1.967238f },
-            { 4448.713379f, 1955.148682f, 441.995178f, 0.380733f },
+            { 4489.819824f, 1955.636108f, 441.996002f, 0.353242f },
```

Comment:
why it was necessary?

Reply:
Revert this. It has nothing to do with the PR

Reply:
reverted